### DISCUSSION TITLE
🦖 Welcome to Dinosaur Exploder Discussions ! 🦖

### DISCUSSION THREAD

**Discussion by dylan-power** [jvondermarck/dinosaur-exploder#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. 

To get started, introduce yourself and let us know anything you wish to discuss.

**Comment by jvondermarck** [comment]
### Any question ? 🦖 
**Feel free to write us something on :** 
- [GitHub Discussion](https://github.com/dylan-power/dinosaur-exploder/discussions)
- [Twitter](https://twitter.com/DinosaurExplod1)
- [Website blog](https://dinosaur-exploder.freecluster.eu/forum)
- [Discord](https://discord.com/invite/nkmCRnXbWm)
- [Mail](dinosaurexploder.conduct@gmail.com)
